src/environment.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <boost/network/protocol/http/client.hpp>
 #include <fstream>
+#include <sstream>
```

Comment:
Naive c++ question, where is this actually being used? Is this in the `std` namespace when you use `std::istringstream in(kube_env);`?

Reply:
Yeah, this is the header for `std::istringstream`.

Reply:
remove this import now that std::istringstream is no longer used.

Reply:
Good catch. Done.